Cubical/Algebra/Module/Instances/FinMatrix.agda

```diff
@@ -0,0 +1,39 @@
+{-# OPTIONS --safe #-}
+module Cubical.Algebra.Module.Instances.FinMatrix where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.SIP
+
+open import Cubical.Functions.FunExtEquiv using (funExt₂)
+
+open import Cubical.Data.Nat using (ℕ)
+
+open import Cubical.Algebra.Matrix
+open import Cubical.Algebra.Ring
+open import Cubical.Algebra.Module
+
+module _ {ℓ} (R : Ring ℓ) {m n : ℕ} where
+
+  open module R' = RingStr (str R) renaming (_+_ to _+r_; -_ to -r_)
+  open LeftModuleStr
+
+  FinVecLeftModule : LeftModule R ℓ
+  fst FinVecLeftModule = FinMatrix ⟨ R ⟩ m n
+  0m  (snd FinVecLeftModule) = λ _ _       → 0r
+  _+_ (snd FinVecLeftModule) = λ xs ys x y → xs x y +r ys x y
+  -_  (snd FinVecLeftModule) = λ xs x y    → -r xs x y
+  _⋆_ (snd FinVecLeftModule) = λ r xs x y  → r · xs x y
+  isLeftModule (snd FinVecLeftModule)    = isLeftModuleR
```

Comment:
'isLeftModuleR' sounds like R is a left module, but it is about something being an R-left-module. So maybe 'isRLeftModule' is better?

Reply:
I have already changed the name.